src/Data.CommandRepositories/Class1.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace PetProjects.MicroTransactionsApi.Data.CommandRepositories
```

Comment:
We won't access the DB as a WriteModel, I think this is unnecessary. The command should be sent to Kafka, and the command handler will take care of inserting in the database.

This API should only have read-only access to the Database right?

Reply:
Yes, in our specific case we won't be needing this csproj. I think we should implement MessageBus logical components in Infrastructure layer and the command handlers would use these components to publish the commands.

Reply:
Yep can be done that way.

Reply:
The way our framework kafka is done, you can inject a producer in the ctor and use it right away.

Reply:
Yeah I have to check out the framework kafka to see if it even makes sense to create Bus related components in the infrastructure layer.